backend/music_app/urls.py

```diff
@@ -28,4 +28,8 @@
     path('api/logout/', views.logout, name='logout'),
     path('api/follow/<int:user_id>/', views.follow_user, name='follow_user'),
     path('api/unfollow/<int:user_id>/', views.unfollow_user, name='unfollow_user'),
+    path('api/create-post/', views.create_post, name='create_post'),
+    path('api/get-posts/', views.get_posts, name='get_posts'),
+    path('api/get-post/<int:post_id>/', views.get_post, name='get_post'),
```

Comment:
I'd do 

POST posts
GET posts -> a single endpoint which can be used like:

/posts?start=<int>&size=<int>

or

/posts?id=<int>